### ISSUE TITLE
Safari error loading runtime-mngr 

### ISSUE THREAD

**Issue by mwfarb** [conix-center/ARENA-core#25]
On Safari 13.1.12 (latest?), the console logs errors loading some runtime components with a double path:
- error: /runtime-mngr/runtime-mngr/index.js.map
- correct: /runtime-mngr/index.js.map

**Comment by nampereira** [comment]
Yeah, this is a js bundler config issue. Did not pay too much attention because the .map file is not really vital for the code to work. Will look into it as low prio.
